pkg/liquidity-source/uniswap-v1/constant.go

```diff
@@ -0,0 +1,25 @@
+package uniswapv1
+
+import "github.com/holiman/uint256"
+
+const (
+	DexType = "uniswap-v1"
+)
+
+var (
+	defaultGas = Gas{Swap: 60000}
```

Comment:
```suggestion
	defaultGas = Gas{Swap: 160000}
```

[go to etherscan to check how much a simple swap directly with the pool costs](https://etherscan.io/tx/0xe2af39ba92f93f975b8f58c480e550ac160f0d1c23ec48f3226b3750da7c26ad)

Reply:
I thought `BaseGas` was already 125000, with a total of 185000 (close to the result in this link, which is 180000)?

Reply:
BaseGas is for our router swap, which includes router logic plus pool swaps. The tx is a call directly to pool